core/templates/dev/head/domain/collection/ReadOnlyCollectionBackendApiServiceSpec.js

```diff
@@ -13,11 +13,11 @@
 // limitations under the License.
 
 /**
- * @fileoverview Unit tests for CollectionBackendApiService.
+ * @fileoverview Unit tests for ReadOnlyBackendApiService.
  */
 
-describe('Collection backend API service', function() {
-  var CollectionBackendApiService = null;
+describe('Read only collection backend API service', function() {
+  var ReadOnlyBackendApiService = null;
```

Comment:
Prefer calling this ``ReadOnlyCollectionBackendApiService`` to maintain parity with the actual service name.